go/pkg/bertyprotocol/contact_request_manager.go

```diff
@@ -316,7 +315,7 @@ func (c *contactRequestsManager) incomingHandler(stream network.Stream) {
 
 func (c *contactRequestsManager) performSend(otherPK crypto.PubKey, stream network.Stream) error {
 	defer func() {
-		if err := p2phelpers.FullClose(stream); err != nil {
+		if err := ipfsutil.FullClose(stream); err != nil {
```

Comment:
This is only useful if you actually want to do something with the error (e.g., return it to indicate that the send didn't complete). This will wait a full round-trip.

If you just want to send, close, then walk away, just call `Close`.